internal/dag/dag.go

```diff
@@ -293,6 +293,16 @@ func (s *Secret) Data() map[string][]byte {
 	return s.Object.Data
 }
 
+// Cert returns the secret's tls certificate
+func (s *Secret) Cert() []byte {
+	return s.Object.Data[v1.TLSCertKey]
+}
+
+// PrivateKey returns the secret's tls private key
+func (s *Secret) PrivateKey() []byte {
+	return s.Object.Data[v1.TLSPrivateKeyKey]
```

Comment:
Same here if the key is nil?

Reply:
Agreed. if the key doesn't exist, the return would be populated as and empty `[]byte`.

Reply:
:+1: